js/test/date/testclock.js

```diff
@@ -1221,7 +1221,7 @@ module.exports.testclock = {
         var info = new LocaleInfo("en-CN");
         test.ok(info !== null);
 
-        test.equal(info.getClock(), 12);
+        test.equal(info.getClock(), 24);
```

Comment:
I vaguely remember the LG China office asked us to change it to 12 hour because of this...

Reply:
If cldr says, the default clock ks 24, shouldn't we follow it?It'w weird   how the current converting tool generate zh/clock.jf file as havine 12 so far. because cldr says 24.
If LG want to have different value than cldr, we can handle in custom locale data layer.